bundles/org.openhab.binding.hpprinter/src/main/java/org/openhab/binding/hpprinter/internal/HPPrinterHandler.java

```diff
@@ -155,7 +155,7 @@ protected ThingBuilder editThing() {
     }
 
     @Override
-    protected void updateProperties(final Map<String, String> properties) {
+    protected void updateProperties(final @Nullable Map<String, String> properties) {
         super.updateProperties(properties);
```

Comment:
In case the overridden methods just coll the super method without any other individual logic I tend to remove them completely. Wdyt?

Reply:
That‘s not possible. `protected` is visible to sub-classes and other classes in the same package, but not to classes in the same package of the sub-class. I’m this case another class in the package of the sub-class needs access and therefore it is necessary to override it. I didn‘t think about that, too.

Reply:
I expected that answer and I have the same feeling that it will be clearer if the overriding method uses `public` modifier. But there is no guideline how to do it.